automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -77,26 +79,86 @@ class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with W
       val user = UserPool.chooseProjectOwner
       implicit val authToken: AuthToken = user.makeAuthToken()
       withCleanBillingProject(user) { projectName =>
+        val methodName = uuidWithPrefix("test_JSON_populate")
+        val method = Method(
+          methodName = methodName,
+          methodNamespace = "MethodConfigSpec",
+          snapshotId = 1,
+          rootEntityType = "sample",
+          synopsis = "",
+          documentation = "",
+          payload = """
+                      |workflow w {
+                      |  call t
+                      |}
+                      |
+                      |task t {
+                      |  String inString
+                      |  Float inFloat
+                      |  Int inInt
+                      |  Boolean inBoolean
+                      |  File inFile
+                      |  Array[String] inStringArray
+                      |  Array[String] inStringArray2
+                      |  Map[String,String] inStringMap
+                      |
+                      |  command {
+                      |    echo inString ${inString}
+                      |    echo inFloat ${inFloat}
+                      |    echo inInt ${inInt}
+                      |    echo inBoolean ${inBoolean}
+                      |    echo inFile ${inFile}
+                      |    echo inStringArray ${sep=" " inStringArray}
+                      |    echo inStringArray2 ${sep=" " inStringArray2}
+                      |    echo inStringMap ${write_map(inStringMap)}
+                      |  }
+                      |  output {
+                      |    String out = stdout()
+                      |  }
+                      |  runtime {
+                      |    docker: "ubuntu"
+                      |  }
+                      |}
+                      |""".stripMargin)
+        api.methods.createMethod(method.creationAttributes)
```

Comment:
For cleanliness, can we delete the method after the test?

Reply:
Best we can do is redact, but I can add that.